src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -788,12 +805,7 @@ private void CreateWindow()
 
             Handle = new WindowImplPlatformHandle(this);
 
-            _multitouch = Win32Platform.Options.EnableMultitouch ?? true;
-
-            if (_multitouch)
-            {
-                RegisterTouchWindow(_hwnd, 0);
-            }
+            RegisterTouchWindow(_hwnd, 0);
```

Comment:
This call is required for both WM_TOUCH and WM_POINTER. EnableMultitouch is obsolete now so I removed old condition.